spec/models/defence_organisation_spec.rb

```diff
@@ -1,18 +1,29 @@
 # frozen_string_literal: true
 
 RSpec.describe DefenceOrganisation, type: :model do
-  subject(:defendant) { described_class.new(body: defence_organisation_hash) }
+  subject(:defence_organisation) { described_class.new(body: defence_organisation_data) }
 
-  let(:defence_organisation_hash) do
-    JSON.parse(file_fixture("defence_organisation.json").read)
+  context "with required fields" do
+    let(:defence_organisation_data) { JSON.parse(file_fixture("defence_organisation/required_fields.json").read) }
+
```

Comment:
`it { expect(defence_organisation.name.id).to be_nil }`

Reply:
another good spot, thanks!